_posts/test1.md

```diff
@@ -2,3 +2,9 @@
 layout:  test pull request
 title:  again
 ---
+another chart
+
+---
```

Comment:
for line 7 ,whether it is a wrong format?i don't know~maybe it only could have a chart in a file?